src/frequenz/client/microgrid/_client.py

```diff
@@ -12,9 +12,12 @@
 
 from frequenz.api.microgrid.v1 import microgrid_pb2_grpc
 from frequenz.client.base import channel, client, retry, streaming
+from google.protobuf.empty_pb2 import Empty
 from typing_extensions import override
 
 from ._exception import ClientNotConnected
+from ._microgrid_info import MicrogridInfo
+from ._microgrid_info_proto import microgrid_info_from_proto
 from .id import ComponentId
 
 DEFAULT_GRPC_CALL_TIMEOUT = 60.0
```

Comment:
This feels like a lot for a client that can only be accessed from a local network.  Was it always like this?

Reply:
If you mean the 60 seconds timeout, yes [it was like this before too](https://github.com/frequenz-floss/frequenz-client-microgrid-python/blob/v0.x.x/src/frequenz/client/microgrid/_client.py#L45).